libs/Texture/Texture.cpp

```diff
@@ -15,7 +16,7 @@ Texture::~Texture()
 
 void Texture::Bind(GLenum TextureUnit)
 {
-	glActiveTexture(GL_TEXTURE0 + TextureUnit);
+	glActiveTexture(GL_TEXTURE0 + 0);
```

Comment:
why add 0? shouldn't that be a no-op?

Reply:
is the argument to this method still needed? seems like it's unused

Reply:
No longer needed. I removed it.